src/pages/permissions/panels/EditGroupPanel.tsx

```diff
@@ -11,26 +16,128 @@ import { renameGroup, updateGroup } from "api/auth-groups";
 import { queryKeys } from "util/queryKeys";
 import { testDuplicateGroupName } from "util/permissionGroups";
 import NotificationRow from "components/NotificationRow";
-import { LxdGroup } from "types/permissions";
+import { LxdGroup, LxdIdentity } from "types/permissions";
 import ScrollableContainer from "components/ScrollableContainer";
+import classnames from "classnames";
+import EditIdentitiesForm, {
+  FormIdentity,
+} from "pages/permissions/panels/EditIdentitiesForm";
+import { getIdentityIdsForGroup } from "util/permissionIdentities";
+import EditGroupPermissionsForm, {
+  FormPermission,
+} from "pages/permissions/panels/EditGroupPermissionsForm";
+import { enrichPermissionId } from "util/permissions";
+import { fetchIdentities, updateIdentities } from "api/auth-identities";
+import LoggedInUserNotification from "pages/permissions/panels/LoggedInUserNotification";
+import { useSettings } from "context/useSettings";
+import { pluralize } from "util/instanceBulkActions";
+import GroupHeaderTitle from "pages/permissions/panels/GroupHeaderTitle";
+import { GroupSubForm } from "pages/permissions/panels/CreateGroupPanel";
 
 interface Props {
   group: LxdGroup;
+  onClose?: () => void;
 }
 
-const EditGroupPanel: FC<Props> = ({ group }) => {
+const EditGroupPanel: FC<Props> = ({ group, onClose }) => {
   const panelParams = usePanelParams();
   const notify = useNotify();
   const toastNotify = useToastNotification();
   const queryClient = useQueryClient();
   const controllerState = useState<AbortController | null>(null);
+  const [subForm, setSubForm] = useState<GroupSubForm>(null);
+  const [confirming, setConfirming] = useState(false);
+  const [identities, setIdentities] = useState<FormIdentity[]>(
+    getIdentityIdsForGroup(group).map((id) => ({ id: id })) as FormIdentity[],
```

Comment:
this is a little unfortunate, but I think it's worth the simplification everywhere else...